config.js

```diff
@@ -99,7 +99,12 @@ config.mqtt = {
     /**
      * Time to keep connection open between client and MQTT broker (default is 0 seconds)
      */
-    keepalive: 0
+    keepalive: 0,
+
+    /**
+     * Whether to use slashes at the beginning of topic when sending or not
+     */
+    avoidLeadingSlashes: false
```

Comment:
```suggestion
    avoidLeadingSlash: false
```

(as in the IOTA-UL PR)

Once applied, I understand this change would need fixes in some other places.

Reply:
Or maybe it's a typo just in this place... (other parts of the PR use the singular form)

Reply:
My bad. Definitely a typo. Thanks btw

Reply:
Fixed in df397c4